students/gene/project/test_node_db.py

```diff
@@ -0,0 +1,38 @@
+from node_db import Node, NodeStorage
+import pytest
+
+
+
+
+@pytest.fixture
+def sample_data():
+
+    ds = NodeStorage()
```

Comment:
I would pass :memory: directly in here -- as a fixture you want to be real sure you're making a clean DB that won't mess anything else up.